dev/workshop/typings/dom.d.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-var */
+
+/**
+ * @todo Remove when TypeScript supports this
```

Comment:
is there a github issue tracking support for these? if so, would be great to include here.